.travis.yml

```diff
@@ -17,14 +17,15 @@ addons:
             - clang-format-3.6
             - libgmp-dev
             - gfortran
-            - libopenblas-dev
             - liblapack-dev
+            - libatlas-base-dev
+            - libatlas-dev
```

Comment:
Because we now use pip install -U numpy to get the most recent version
of numpy, and this requires this headers to compile correctly.